src/test/kotlin/no/nav/familie/tilbake/kravgrunnlag/AutomatiskBehandlingAvKravgrunnlagUnder4RettsgebyrTest.kt

```diff
@@ -172,4 +172,35 @@ class AutomatiskBehandlingAvKravgrunnlagUnder4RettsgebyrTest : OppslagSpringRunn
             ),
         )
     }
+
+    private fun assertFakta() {
+        val behandlingsstegstilstand = behandlingsstegstilstandRepository.findByBehandlingId(behandlingId)
+        behandlingsstegstilstand.any { it.behandlingssteg == Behandlingssteg.IVERKSETT_VEDTAK } shouldBe true
+        val fakta = faktaFeilutbetalingRepository.findFaktaFeilutbetalingByBehandlingIdAndAktivIsTrue(behandlingId)
+        fakta.begrunnelse shouldBe Constants.AUTOMATISK_SAKSBEHANDLING_UNDER_4X_RETTSGEBYR_FAKTA_BEGRUNNELSE
+    }
+
+    private fun sendVedtaksbrev() {
+        val sendØkonomiTilbakekrevingsvedtak = taskService.finnAlleTaskerMedPayloadOgType(behandlingId.toString(), SendØkonomiTilbakekrevingsvedtakTask.TYPE).first()
+        sendØkonomiTilbakekrevingsvedtakTask.onCompletion(sendØkonomiTilbakekrevingsvedtak)
```

Comment:
Potensiell forbedring/sikkerhet: 
For å slippe å kjøre onCompletion selv kan vi trigge taskWorker til å plukke taskene istedenfor

Reply:
Vi vurderer å lage dette (neste pr)

Reply:
Synes kanskje det blir litt lite isolert for en enhetstest? Kan eventuelt lage en egen test som kjører gjennom alt, hvor taskWorker plukker tasker etc.